src/storage/hummock_test/src/test_utils.rs

```diff
@@ -206,3 +207,11 @@ pub async fn prepare_local_version_manager_new(
 
     (local_version_manager, event_tx, event_rx)
 }
+
+/// Prefix the `key` with table id 0.
+pub fn prefixed_key<T: AsRef<[u8]>>(key: T) -> Bytes {
+    let mut buf = Vec::new();
+    buf.put_u32(0);
```

Comment:
We may use a test table id instead of 0.

Reply:
What should the test table id be? 🤔

Reply:
Something other than 0. `TEST_TABLE_ID = 233u32` may be fine.

Currently we use `TableId::default()` a lot in our unit tests and have many hacks `table_id == 0` in our code to pass the unit tests. Maybe we can remove these hack and pass a more meaningful table id.